content/2021-losangeles/scholarship.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Registration Discounts/Scholarships
+---
+
+To encourage and increase the participation of a diverse community, we are pleased to offer diversity and need-based registration scholarships, discounts, and travel funding to our conferences.
+
+[**APPLY NOW**](https://www.surveymonkey.com/r/Co-LoScholarshipNA21)
+
+## Diversity Scholarships
+
+Eligibility Requirements: Applicants must be from traditionally underrepresented and/or marginalized groups in the technology and/or open source communities who are unable to attend without some financial assistance. Including, but not limited to persons identifying as: LGBTQIA, women, persons of color, and/or persons with dis/abilities. (Virtual or In-Person)
+
+## Need-Based Scholarships
+
+Eligibility Requirements:  Applicants should be active community members who are not being assisted or sponsored by a company or organization, and are unable to attend for financial reasons. (Virtual or In-Person)
+
+Scholarships are awarded based on a combination of need and impact. Scholarships are NOT transferable to another person, or any other event. Selection will be made by an assembled group of reviewers who will assess each applicant’s request. Application information will be kept confidential.
+
+This is a Hybrid Event. The in-person event will take place in Los Angeles, CA. The virtual event will be accessible on-line.
+
+Virtual Scholarship recipients will receive a complimentary registration pass to attend KubeCon+CloudNativeCon North America 2021 Virtual (on-line), as well as one CNCF-hosted Co-located Event (on-line).
+
+In-Person Scholarship recipients will receive a complimentary registration pass to attend KubeCon+CloudNativeCon North America 2021 in Los Angeles, including one CNCF-hosted Co-located Event and up to a max of $1,500 USD to reimburse actual travel expenses. Please note that reimbursements are made after the event, and the stipend may only be used for the following: coach airfare, hotel / accommodations (up to two nights), transportation to and from the airport, and visa application fee.
+
+Receipt of a scholarship does not, on its own, guarantee entry to the event. Recipients need to adhere to The Linux Foundation COVID-19 rules and regulations. If the recipient does not meet COVID-19 requirements for the event, the scholarship is null and void. Please Note: Attendance at KubeCon + CloudNativeCon North America 2021 and co-located in-person events requires proof of vaccination. To learn more, please [click here](https://www.linuxfoundation.org/blog/adoption-of-a-covid-19-vaccine-required-approach-for-our-fall-2021-event-line-up/).
+
+## Registration Scholarship Applications Due:
+
+In-Person: Sunday, August 8, 2021 (11:59PM Pacific Time)
+Virtual: Sunday, September 26, 2021 (11:59PM Pacific Time)
+
+## All Applicants Will be Notified By:
```

Comment:
This is the only page where we Caps all title, can we uncap? Thanks.

Reply:
Yes That Is A Good Point Thanks